pkg/metadata/annotations.go

```diff
@@ -27,9 +27,10 @@ const (
 
 	AnnotationPrefix = "pulumi.com/"
 
-	AnnotationAutonamed      = AnnotationPrefix + "autonamed"
-	AnnotationSkipAwait      = AnnotationPrefix + "skipAwait"
-	AnnotationTimeoutSeconds = AnnotationPrefix + "timeoutSeconds"
+	AnnotationAutonamed        = AnnotationPrefix + "autonamed"
+	AnnotationSkipAwait        = AnnotationPrefix + "skipAwait"
+	AnnotationTimeoutSeconds   = AnnotationPrefix + "timeoutSeconds"
+	AnnotationCreateApiVersion = AnnotationPrefix + "createApiVersion"
```

Comment:
Hmm. Might be worth bike-shedding here. `initialApiVersion`?

Reply:
No strong opinion. I'm fine changing it.